### PR TITLE
Update step quote verification

### PR DESCRIPTION
**Pull request by wer1st**
Now it accepted for `DiscreteQuotation` to have some slippage between chunk prices, not the strong order.

### CODE REVIEW

**Comment by wer1st** [comment]
Was moved into #1141

File: pallets/liquidity-proxy/src/liquidity_aggregator/selector.rs [hunk 1]
```diff
@@ -69,10 +68,8 @@ where
         &mut self,
         source: LiquiditySourceType,
         discrete_quotation: DiscreteQuotation<AssetIdOf<T>, Balance>,
```
Comment (vovac12) [inline pallets/liquidity-proxy/src/liquidity_aggregator/selector.rs:70]: Maybe call this method `add_source_unchecked`?

Comment (wer1st) [reply pallets/liquidity-proxy/src/liquidity_aggregator/selector.rs:70]: For what?

Comment (vovac12) [reply pallets/liquidity-proxy/src/liquidity_aggregator/selector.rs:70]: To show that this function does not perform any checks before adding liquidity and it's caller's responsobility to perform any checks

Comment (wer1st) [reply pallets/liquidity-proxy/src/liquidity_aggregator/selector.rs:70]: It makes sense only when we have another "checked" function.

**Review by vovac12 — APPROVED** [review]

**Review by Alexey-N-Chernyshov — APPROVED** [review]

**Pull request merged**